hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azure/NativeAzureFileSystem.java

```diff
@@ -3549,6 +3550,90 @@ public void setOwner(Path p, String username, String groupname)
     }
   }
 
+  /**
+   * Set the value of an attribute for a path.
+   *
+   * @param path The path on which to set the attribute
+   * @param xAttrName The attribute to set
+   * @param value The byte value of the attribute to set (encoded in utf-8)
+   * @param flag The mode in which to set the attribute
+   * @throws IOException If there was an issue setting the attribute on Azure
+   */
+  @Override
+  public void setXAttr(Path path, String xAttrName, byte[] value, EnumSet<XAttrSetFlag> flag) throws IOException {
```

Comment:
too many line breaks in this method. Please remove the extra ones.

Reply:
Done in 54dcb1d.

Reply:
Re-activating this comment as i don't see this fixed.

Reply:
I removed some more newlines in 6ecbdc0. I'm finding it a bit difficult to discern what style to follow since the file seems to include multiple styles for newlines (e.g. my original implementation was close to `setOwner` in indentation style). If you have any additional feedback on-top of what is implemented now, I'd appreciate more concrete pointers on style/formatting. Thanks in advance!

Reply:
Sorry about nit-picking but it would be good to just follow the formatting in other methods in this file. For example, you don't need new lines in the catch block and also remove 3588.

Reply:
Thanks for the pointers. I was previously following the style of [setOwner](https://github.com/apache/hadoop/blob/29049e6fee271298f28ff2c484c940527370c96d/hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azure/NativeAzureFileSystem.java#L3498-L3551). I removed the newlines you mentioned in f7c0d26.